src/main/java/com/it4045/pokemonviewer/enterprise/dto/enums/PokemonType.java

```diff
@@ -0,0 +1,21 @@
+package com.it4045.pokemonviewer.enterprise.dto.enums;
+
+public enum PokemonType {
+    ONIX,
+    DARK,
+    STEEL,
+    ICE,
+    FIGHTING,
+    FIRE,
+    GROUND,
+    FLYING,
+    PSYCHIC,
+    POISON,
+    ELECTRIC,
+    WATER,
+    BUG,
+    FAIRY,
+    GHOST,
+    GRASS,
+    SALAMENCE
+}
```

Comment:
This abstraction will provide you a place to store all of the PokemonTypes. Sorry if the list is incomplete or incorrect. I researched pokemon types on google and this is what I found.